build-tools/xaprepare/xaprepare/Steps/Step_PrepareBundle.cs

```diff
@@ -85,7 +85,7 @@ protected override async Task<bool> Execute (Context context)
 				}
 
 				Log.DebugLine ("Moving unpacked bundle from {tempDir} to {Configurables.Paths.Bundle_InstallDir}");
-				Utilities.MoveDirectoryContentsRecursively (tempDir, Configurables.Paths.BundleInstallDir, resetFileTimestamp: true);
+				Utilities.MoveDirectoryContentsRecursively (tempDir, Configurables.Paths.BundleInstallDir, resetFileTimestamp: true, ignoreDeletionErrors: true);
```

Comment:
Yeah as @pjcollins mentioned, maybe we don't need this `ignoreDeletionErrors` option.

Reply:
If we don't have it then we might not be able to unpack the bundle. I've changed the default.